web_poet/fields.py

```diff
@@ -94,24 +94,27 @@ def __get__(self, instance, owner=None):
             return self.unbound_method(instance)
 
         @staticmethod
-        def _process(value):
-            for processor in out:
-                value = processor(value)
+        def _process(value, page_object):
+            if hasattr(page_object, "validate_input"):
```

Comment:
Is this check needed? It seems it's run twice, as `_validate_input` checks the same.